### ISSUE TITLE
Add bike lane data for other jurisdictions

### ISSUE THREAD

**Issue by alulsh** [dcfemtech/hackforgood-waba-map#10]
Right now we only have bike lane data for Washington, D.C. proper. We need to add bike lane data for the following jurisdictions covered by WABA. 

Who wants to find and add the data for each jurisdiction? We can split up the work.

- [x] Arlington County, VA - @BuffyShepard
- [x] Alexandria City, VA - @BuffyShepard
- [ ] Fairfax County, VA - @sara4844 
- [x] Montgomery County, MD - @sara4844 
- [x] Prince George's County, MD - @kimcabb

**Comment by kimcabb** [comment]
I will look at finding bike lane data for Prince George's County.

**Comment by evastern** [comment]
I've got links on Hackpad that I'm pulling over I've identified most of near to DC Virginia counties, and Montgomery County.

**Comment by kimcabb** [comment]
A Collection of web map services and other bicycle-related data for the Metropolitan Washington region.

Bicycle Facilities of the NCR
https://www.arcgis.com/home/item.html?id=b42a9e5e317b48059889c555723eb066

**Comment by alulsh** [comment]
@sara4844 added the Montgomery County bike lanes in https://github.com/dcfemtech/hackforgood-waba-map/pull/12!

**Comment by kimcabb** [comment]
As an update: 

The PG County data are limited; finding an API for geoJSON format is not forthcoming.  Nellie has suggested we move ahead with other MVP goals and advocate for access.  I will submit feedback to OpenPGC. https://data.princegeorgescountymd.gov/

**Comment by alulsh** [comment]
Arlington County and Alexandria City GeoJSON data added in https://github.com/dcfemtech/hackforgood-waba-map/pull/17 by @BuffyShepard.

Fairfax County bike lane data added by @sara4844 in https://github.com/dcfemtech/hackforgood-waba-map/pull/19.

**Comment by alulsh** [comment]
@alongthepike has a PR open (https://github.com/dcfemtech/hackforgood-waba-map/pull/62) to add PG county. @alongthe pike also filtered the Montgomery County data to remove the sharrows.

Once this is merged we can close this ticket - we should open a separate issue for Fairfax County bike data.

**Comment by alulsh** [comment]
@alongthepike added PG county data in https://github.com/dcfemtech/hackforgood-waba-map/pull/71 and it was added to our front end in https://github.com/dcfemtech/hackforgood-waba-map/pull/83.

Going to close this in favor of a dedicated issue for Fairfax County data.
